### ISSUE TITLE
Mac new Outlook crash when try to support event-based activation

### ISSUE THREAD

**Issue by hello-chenchen** [OfficeDev/office-js#3756]
here is the crash report.
[Outlook crash.log](https://github.com/OfficeDev/office-js/files/12894616/Outlook.crash.log)

**Comment by exextoc** [comment]
@hello-chenchen 

- For which launch event did you face this issue?
- Could you please share the repro steps ?

**Comment by hello-chenchen** [comment]
@exextoc here is the launch event configure.

```
<ExtensionPoint xsi:type="LaunchEvent">
                  <LaunchEvents>
                    <LaunchEvent Type="OnAppointmentSend" FunctionName="OnAppointmentSendFunc" SendMode="PromptUser"/>
                    <LaunchEvent Type="OnAppointmentOrganizer" FunctionName="OnAppointmentOrganizerFunc"/>
                  </LaunchEvents>
                  <SourceLocation resid="WebViewRuntimeUrl"/>
                </ExtensionPoint>
```

and reproduce is create an recurrence event, then edit an occurrence event. will be crash in some time.

**Comment by exextoc** [comment]
@hello-chenchen the logs indicate you are using version 16.75. Please update to the latest version and let us know if the issue persists.

**Comment by hello-chenchen** [comment]
@exextoc , so can you confirm that your engineer have find the root cause in 16.75, and have fixed this problem in latest version?

**Comment by exextoc** [comment]
@hello-chenchen We have fixed multiple threading issues similar to the one mentioned here in the old version. Please update to the latest version and confirm, so that we can identify if this a different issue.

**Comment by hello-chenchen** [comment]
hi @exextoc , please reopen it. we still can reproduce it in latest mac outlook version.(16.78.3, which is updated in 2023.11.01)
[outlook crash 1113-02.docx.zip](https://github.com/OfficeDev/office-js/files/13346186/outlook.crash.1113-02.docx.zip)

**Comment by exextoc** [comment]
We have fixed this issue and fix is available in version >= 16.80 (23121017)

**Comment by hello-chenchen** [comment]
hi @exextoc , yes, we have verified. it's worked.
